src/components/DeleteConfirmationDialog.js

```diff
@@ -8,24 +8,15 @@ import { deleteDive } from "../redux/actions";
 import firebase from 'firebase/app';
 
 const flex_row_style = {
-  display: "flex",
-  flexDirection: "row",
-  paddingTop: "1em",
```

Comment:
deleting the css functionality and not changing the html will break the dialog layout completely.